cluster/up.sh

```diff
@@ -20,10 +20,12 @@ case "${KUBEVIRT_PROVIDER}" in
         ;;
 esac
 
+CREATE_SECOND_NIC="-device virtio-net-pci,netdev=network2 -netdev tap,id=network2,ifname=tap2,script=no,downscript=no"
+
 echo "Install cluster from image: ${image}"
 if [[ $image == $KUBERNETES_IMAGE ]]; then
     # Run Kubernetes cluster image
-    ./cluster/cli.sh run --random-ports --nodes ${KUBEVIRT_NUM_NODES} --memory ${KUBEVIRT_MEMORY_SIZE} --background kubevirtci/${image}
+    ./cluster/cli.sh run --random-ports --nodes ${KUBEVIRT_NUM_NODES} --memory ${KUBEVIRT_MEMORY_SIZE} --background --qemu-args "'${CREATE_SECOND_NIC}'" kubevirtci/${image}
```

Comment:
you should probably add in the reame.md what value should CREATE_SECOND_NIC should be set to

Reply:
done, could you please take another look?